src/components/visualizations/chart/highcharts/comboConfiguration.ts

```diff
@@ -23,23 +24,27 @@ export function getDefaultChartType(config?: IChartConfig) {
     return LINE;
 }
 
+function isLineLineCombo(config?: IChartConfig) {
+    const primaryType = get(config, "primaryChartType", COLUMN);
+    const secondaryType = get(config, "secondaryChartType", LINE);
```

Comment:
we can avoid using get, it returns `any`
```
const primaryType = config.primaryChartType || COLUMN;
const secondaryType = config.secondaryChartType || LINE;
```

Reply:
Use lodash/get here is acceptable. because `config` is optional.

Reply:
so could it have default value?